client/src/connection/com/script.ts

```diff
@@ -0,0 +1,108 @@
+export const scriptContent = `
+class SASRunner{
+  [System.__ComObject] $objSAS
+  
+  [void]ResolveSystemVars(){
+  $code =
+@'
+   %let workDir = %sysfunc(pathname(work));
+   %put &=workDir;
+   %let rc = %sysfunc(dlgcdir("&workDir"));
+   run; 
+'@
+    $this.Run($code)
+    $varLogs = $this.FlushLog(4096)
+    Write-Host $varLogs
+  }
+  [void]Setup([string]$profileHost) {
+    try {
+        # create the Integration Technologies objects
+        $objFactory = New-Object -ComObject SASObjectManager.ObjectFactoryMulti2
+        $objServerDef = New-Object -ComObject SASObjectManager.ServerDef
+        $objServerDef.MachineDNSName = $profileHost # SAS Workspace node
+        $objServerDef.Port = 0  # workspace server port
+        $objServerDef.Protocol = 0     # 0 = COM protocol
+        
+        # Class Identifier for SAS Workspace
+        $objServerDef.ClassIdentifier = "440196d4-90f0-11d0-9f41-00a024bb830c"
+
+        # create and connect to the SAS session 
+        $this.objSAS = $objFactory.CreateObjectByServer(
+            "SASApp", # server name
```

Comment:
I know it doesn't technically matter, but using "SASApp" as the name makes me a little uncomfortable, since that is the default name for SAS Workspace Servers. Perhaps we could use "Local" instead?

Reply:
```suggestion
            "Local", # server name
```